tests/integration/cmor/_fixes/icon/test_icon.py

```diff
@@ -1607,6 +1607,9 @@ def test_invalid_time_units(cubes_2d):
 # Test fix with (sub-)hourly data
 
 
+# remove xfail when https://github.com/pandas-dev/pandas/issues/57002
+# gets fixed; pinning pandas not a viable solution due deps issues
+@pytest.mark.xfail(reason='Bug in pandas needs be fixed.')
```

Comment:
As a short-term solution this is fine (it shouldn't break any recipes), but a proper fix would be much nicer.